pkg/pxc/statefulset.go

```diff
@@ -121,13 +123,13 @@ func PodAffinity(af *api.PodAffinity, app api.App) *corev1.Affinity {
 	return nil
 }
 
-func getConfigVolumes() corev1.Volume {
+func getConfigVolumes(cr *api.PerconaXtraDBCluster) corev1.Volume {
 	vol1 := corev1.Volume{
 		Name: "config-volume",
 	}
 
 	vol1.ConfigMap = &corev1.ConfigMapVolumeSource{}
-	vol1.ConfigMap.Name = appName
+	vol1.ConfigMap.Name = strings.Join([]string{cr.Name, appName}, "-")
```

Comment:
I don't really like an idea to "create" a ConfigMap Name in two different places (here and https://github.com/Percona-Lab/percona-xtradb-cluster-operator/pull/71/files#diff-b06a534087720544611c913dcd53a544R19). Moreover, if to add config map to the proxy - this pattern will broke.
My suggestion is to use a StatefulSet label `component` for the name. And pass it like parameter to both functions (`getConfigVolumes` and `NewConfigMap`).
For the labels see: https://github.com/Percona-Lab/percona-xtradb-cluster-operator/blob/master/pkg/apis/pxc/v1alpha1/perconaxtradbcluster_types.go#L136